labs/TrapheusAI/trapheusai_app.py

```diff
@@ -47,6 +47,9 @@ def handle_dataset_search():
                 repository_url = data["fullurl"]
                 df = pd.read_csv(repository_url)
                 streamlit.data_editor(df)
+                streamlit.download_button(
```

Comment:
Can we check if we edit the dataset and then download are the changes retained?

Reply:
It won't retain the changes but we can modify this to retain the changes if required.

Reply:
Merging this, can you open a new issue and work on saving the in memory CSV file